test/functional/rpc_scanblockfilters.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test the scantxoutset rpc call."""
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+
+class ScanblockfiltersTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 1
+        self.extra_args = [["-blockfilterindex=1"]]
```

Comment:
Maybe add a node without this index and test that the RPC call fails.

Reply:
and `assert_raises_rpc_error` for "Unknown filtertype", "Index is not enabled for filtertype " + filtertype_name, and "Invalid stopheight"